features/shop/locale/handling_different_locales_on_multiple_channels.feature

```diff
@@ -12,20 +12,21 @@ Feature: Handling different locales on multiple channels
         And that channel allows to shop using "Polish (Poland)" and "Norwegian (Norway)" locales
         And it uses the "Polish (Poland)" locale by default
 
-    @ui @api
+#   These scenarios should be implemented after implementing Order resource in APIP3
```

Comment:
How come?
I don't see anything indicating using anything other than channel and locale in the scenarios.

Reply:
To check the current locale, we are using endpoint for creating an order:
https://github.com/Sylius/Sylius/blob/api-platform-3/src/Sylius/Behat/Context/Api/Shop/LocaleContext.php#L105
it doesn't look like the best solution, but I don't see much of an alternative 🤔